viaMover/web-app
# NFT: Update Dice Project NFT assets

What was done
* Updated dice project NFT assets

How to test
* visit "more" page, check that current source is the new one (with `_more` postfix)
* visit NFT "view all" page, check that current source is the new one (without `_more` postfix)
